### ISSUE TITLE
Dragging window on ngPopup demo causes it to shrink

### ISSUE THREAD

**Issue by keirlawson** [MarkoCen/angular-popup#8]
This is very odd bahviour, seen in both Firefox and Chrome

**Comment by nobukichi** [comment]
making updateElementSize function starting from line 210 empty at least stops the shrinking behaviour. debugging the function should solve the problem. I may spend time to debug it if we decide to use ngPopup in our project. (I am just evaluating it for now.)

**Comment by nobukichi** [comment]
a slightly better fix is to replace updateParentScopeOptions function with following

```

    updateParentScopeOptions: function(options,element){
        var $element = element[0];
        options.position.top =  $element.offsetTop;
        options.position.left = $element.offsetLeft;
        if($element.getElementsByClassName('content')[0].style.display != 'none'){
            options.width = $element.clientWidth + 2;  //! add magical 2
            options.height = $element.clientHeight + 2;   //! add magical 2
        }
    },

```

adding 2 to both width and height. Although it does fix the issue, I still do not understand where this 2 pixels offsets come from. So this fix may not work on other environment where 2 is actually 1 or 3.

**Comment by MarkoCen** [comment]
Sorry for the late response... this issue has been fixed on e1b8f537261b04b7ad81b556f2c32db2ca0c36a5

**Comment by nobukichi** [comment]
I tried the new build. It does fix the shrinking behaviour, however, the resizing operation never stops once we start resizing by dragging the edge/corner of the widget this time.
